src/styles/colors.ts

```diff
@@ -1,6 +1,6 @@
 export const Colors = {
-  primary: "purple",
+  primary: "black",
```

Comment:
Thank you for your PR. I like black but since we change the bg color to white this won't work. Can we use orange? I think it looks good with a black background.

What do you think?